data/hai/hai reveal 5 philosophers.txt

```diff
@@ -485,8 +485,8 @@ mission "Hai Reveal [B05-long] The Red Mystery"
 
 mission "Hai Reveal [B06-long] Tarazed Analysis"
 	landing
-	name "Seeking Aid"
-	description `Find and board a smugglers ship. Your display will identify them as the "Smuggler" government. Try Syndicate space or the Far North. Then investigate any leads and return to <planet>.`
+	name "Hunting Smugglers"
+	description `Find and board a smugglers ship. Your display will identify them as the "Smuggler" government but they will still look like Merchants. Try Syndicate space or the Far North. Then investigate any leads and return to <planet>.`
```

Comment:
```suggestion
	description `Find and board a smugglers ship. Your display will identify them as the "Smuggler" government but they will still look like Merchants. Try Syndicate space or the Far North, deep into pirate territory. Then investigate any leads and return to <planet>.`
```
this is mostly what I was thinking of (doesnt have to be this exactly, also maybe its just me that finds Far North not clear enough?)
also shouldnt it be `smuggler ship`?

Reply:
![image](https://user-images.githubusercontent.com/60949828/209135297-b67cc1c7-39aa-41df-9088-f8667424c086.png)
Come on Hurl, you can read the government files as well as anyone else.

And explicitly stating pirate territory definitely falls into hand-holding territory.
Besides, in the north they're actually most common in the systems around Alnitak, which is distinctly *not* "deep into pirate territory" - so not only would it be hand-holding, it would also be straight-up incorrect.

Reply:
It's the spawnrates I read and altinak was useless because the navy kept killing them